base_layer/wallet/src/key_manager_service/storage/sqlite_db/key_manager_state.rs

```diff
@@ -150,50 +145,3 @@ pub struct KeyManagerStateUpdateSql {
     branch_seed: Option<String>,
     primary_key_index: Option<Vec<u8>>,
 }
-
-impl Encryptable<XChaCha20Poly1305> for KeyManagerStateSql {
```

Comment:
why have we removed this?
Are we not encrypting the index of the km states anymore?

Reply:
I think there is no need for encrypting this data, but I might be wrong ?